PowerEditor/src/NppIO.cpp

```diff
@@ -626,7 +629,7 @@ void Notepad_plus::doClose(BufferID id, int whichOne, bool doDeleteBackup)
 		}
 
 		if (PathFileExists(buf->getFullPathName()))
-			_lastRecentFileList.add(buf->getFullPathName());
+			fileFullPath += buf->getFullPathName();

```

Comment:
It should be not be an issue as variable is declared with empty string. Let me know if it really needs to be changed.